ibis/tests/util.py

```diff
@@ -47,7 +47,8 @@ def __init__(self):
                                                 'True').lower() == 'true'
         self.use_kerberos = os.environ.get('IBIS_TEST_USE_KERBEROS',
                                            'False').lower() == 'true'
-
+        self.udf_so = self.test_data_dir + '/udf/libudfsample.so'
+        self.udaf_so = self.test_data_dir + '/udf/libudasample.so'
```

Comment:
I'd prefer to eliminate these two from the `IbisTestEnv` object.  Every other field is something that is set from an env var.  I'd rather use it the same way that `test_data_dir` is used in the e2e test file.  In that case, every test that needs it will hardcode the necessary dirs into the test.  An even better way, perhaps, would be to create a new testing class specific to UDFs, and you can have a setup method there that sets the correct `udf_so` directory.